js/src/components/ContentNavigation.js

```diff
@@ -39,44 +43,59 @@ const ContentNavigation = ({
         const buttonProps = {
           variant: 'contained',
           color: 'primary',
+          size: 'large',
           endIcon: <ChevronRightIcon />,
           disabled,
         };
 
         return (
           <LessonGrid>
-            {sections[currentIndex + 1] && (
-              <Button
-                {...buttonProps}
-                onClick={() => history.push({ pathname: `/section-${currentIndex + 2}` })}
-              >
-                {disabled ? completeAnswer : Drupal.t('Next', {}, { context: 'ANU LMS' })}
-              </Button>
-            )}
+            <ButtonWrapper>
+              {noPrevLesson && prevIsLesson && (
```

Comment:
Seems like this condition doesn't cover all cases when "back" button should be displayed.